pyvista/core/utilities/geometric_objects.py

```diff
@@ -1759,8 +1814,7 @@ def CircularArcFromNormal(center, resolution=100, normal=None, polar=None, angle
         normal = [0, 0, 1]
     if polar is None:
         polar = [1, 0, 0]
-    if angle is None:
-        angle = 90.0
+    angle_float = 90.0 if angle is None else angle
```

Comment:
Hungarian Notation is unnecessary in my opinion, as type hints provide sufficient clarity about the type
```suggestion
    angle = 90.0 if angle is None else angle
```

Reply:
`angle` has type `float | None` from the type hint in the signature. But some methods later on in the implementation require only `float` inputs (None is not allowed). So, in order to narrow the type, we need to define a new variable. Here, because of the logic with None, mypy will infer `angle_float` to have type float exclusively, which is what we need.

Alternatively, we can use `cast` to narrow the type, though IMO it's better to avoid using cast (or ignores) wherever possible.

Reply:
Got it. Let's use `angle_`.
```suggestion
    angle_ = 90.0 if angle is None else angle
```